src/sql-editor/standardSql/statementPositionResolversRegistry.ts

```diff
@@ -164,119 +103,76 @@ export function initStatementPositionResolvers(): StatementPositionResolversRegi
     {
       id: StatementPosition.AfterFromKeyword,
       name: StatementPosition.AfterFromKeyword,
-      resolve: (
-        currentToken,
-        previousKeyword,
-        previousNonWhiteSpace,
-        previousIsSlash
-      ) => Boolean(previousNonWhiteSpace?.value.toLowerCase() === FROM),
+      resolve: (currentToken, previousKeyword, previousNonWhiteSpace, previousIsSlash) =>
+        Boolean(previousNonWhiteSpace?.value.toLowerCase() === FROM),
     },
     {
       id: StatementPosition.AfterFrom,
       name: StatementPosition.AfterFrom,
-      resolve: (
-        currentToken,
-        previousKeyword,
-        previousNonWhiteSpace,
-        previousIsSlash
-      ) =>
+      resolve: (currentToken, previousKeyword, previousNonWhiteSpace, previousIsSlash) =>
         Boolean(
-          (previousKeyword?.value.toLowerCase() === FROM &&
-            previousNonWhiteSpace?.isDoubleQuotedString()) ||
-            (previousKeyword?.value.toLowerCase() === FROM &&
-              previousNonWhiteSpace?.isIdentifier()) ||
-            (previousKeyword?.value.toLowerCase() === FROM &&
-              previousNonWhiteSpace?.isVariable())
+          (previousKeyword?.value.toLowerCase() === FROM && previousNonWhiteSpace?.isDoubleQuotedString()) ||
+            (previousKeyword?.value.toLowerCase() === FROM && previousNonWhiteSpace?.isIdentifier()) ||
+            (previousKeyword?.value.toLowerCase() === FROM && previousNonWhiteSpace?.isVariable())
           //  cloudwatch specific
           // (previousKeyword?.value === SCHEMA && previousNonWhiteSpace?.is(TokenType.Parenthesis, ')'))
         ),
     },
     {
       id: StatementPosition.AfterTable,
       name: StatementPosition.AfterTable,
-      resolve: (
-        currentToken,
-        previousKeyword,
-        previousNonWhiteSpace,
-        previousIsSlash
-      ) => {
+      resolve: (currentToken, previousKeyword, previousNonWhiteSpace, previousIsSlash) => {
         return Boolean(
           previousKeyword?.value.toLowerCase() === FROM &&
             (previousNonWhiteSpace?.isVariable() ||
-              previousNonWhiteSpace?.value !== "")
+              (previousNonWhiteSpace?.value !== '' && previousNonWhiteSpace?.value.toLowerCase() !== FROM))
```

Comment:
the diff is a little hard to see due to some weird formatting, but this is the only difference right? Maybe nice to add a unit test if it's do-able, otherwise seems good to me!

Reply:
yeah sorry for that! now it should be formatted according to our standards. this was the only change btw!

I've added a unit test